src/main/java/it/unitn/disi/smatch/CLI.java

```diff
@@ -183,6 +202,36 @@ public static void main(String[] args) throws IOException, DISIException, ClassN
                         log.error("Not enough arguments for mappingFilter command.");
```

Comment:
Given logging is already there, perhaps moving verbosity to logging would be the way to do it...